trino-lb/src/trino_client/cluster_info.rs

```diff
@@ -34,6 +34,7 @@ pub enum Error {
     },
 }
 
+#[allow(dead_code)]
```

Comment:
question: Why is this needed? Shouldn't we remove dead code instead of allowing it here?

Reply:
This is the HTTP json response we get from the trino cluster when asking for it's state. We current#y don't read all it's fields, but on the other hand I already "reverse engineered" them, so I though there is value in keeping them in case we need them later on. But the compiler is too smart.
No strong opinion on this

Reply:
Nope, that sounds fine. I was just curious. We can leave it as is :)